changelog/cache-generated-test-config.dd

```diff
@@ -0,0 +1,4 @@
+Caching the generated unittest main file.
+
+Prevent dub from rebuilding a unittest on a secondary `dub test` with out file changes.
+For dub projects without a user defined unittest configuration.
```

Comment:
```suggestion
For projects without a user-defined unittest configuration `dub test` generates a main file automatically.
This main file is now being cached and won't be regenerated won subsequent runs without file changes.
```

(though to be fair, this feature won't affect many people as typically you don't rerun the testsuite without making changes to a file)

Reply:
Thank you